pages/advanced/smart-account-modules.mdx

```diff
@@ -8,22 +8,28 @@ Safe Modules add custom features to Safe contracts. They are smart contracts tha
 
 Safe Modules can include daily spending allowances, amounts that can be spent without the approval of other owners, recurring transactions modules, and standing orders performed on a recurring date. For example, paying your rent or social recovery modules may allow you to recover a Safe if you lose access to owner accounts.
 
+<Callout type='error' emoji='‼️'>
+  Safe Modules can be a security risk since they can execute arbitrary
+  transactions. Only add trusted and audited modules to a Safe. A malicious
+  module can take over a Safe.
+</Callout>
+
 ![diagram-safe-modules](../../assets/diagram-safe-modules.png)
 
-## How to create a Safe Module
+## Examples 
 
-A great way to understand how Safe Modules work is by creating one. An excellent place to start is [Safe Modding 101: Create your own Safe Module](https://www.youTube.com/watch?v=nmDYc9PlAic).
+### Official Safe Modules
 
-<YouTube embedId='nmDYc9PlAic' />
+These are audited Safe Modules built and maintained by the Safe team. The [safe-modules](https://github.com/safe-global/safe-modules) repository contains the collection of the Official Safe Modules. 
 
-## Examples
+For example, the 4337 module, the allowance module, and the passkeys module.
```

Comment:
I wouldn't say "for example" here, this is the concrete list (and the recovery module will be added soon).